src/taxes/income-tax.ts

```diff
@@ -496,6 +502,18 @@ export function getProvincialTaxAmount(
     grossIncome: number,
     inflationRate = 0,
     yearsToInflate = 0,
+): number {
+    const baseTaxAmount = getProvincialBaseTaxAmount(province, grossIncome, inflationRate, yearsToInflate);
+    const baseCredit = getProvincialBaseCredit(province, inflationRate, yearsToInflate);
+    const surTax = getProvincialSurtaxAmount(province, baseTaxAmount, inflationRate, yearsToInflate);
+    return Math.max(baseTaxAmount - baseCredit, 0) + surTax;
```

Comment:
Pour calculer la surtax, tu dois prendre `baseTaxAmount - baseCredit`
```suggestion
    const baseTaxAmount = getProvincialBaseTaxAmount(province, grossIncome, inflationRate, yearsToInflate);
    const baseCredit = getProvincialBaseCredit(province, inflationRate, yearsToInflate);
    const tax = Math.max(baseTaxAmount - baseCredit, 0)
    const surTax = getProvincialSurtaxAmount(province, tax, inflationRate, yearsToInflate);
    return tax + surTax;
```

Reply:
Je l'ai rechangé ce matin vers la version que j'ai fait un commit, j'avais une différence de valeur produite par rapport à l'accept. Je vais valider de nouveau.